contracts/L2/gateway/L2Migrator.sol

```diff
@@ -57,6 +57,8 @@ contract L2Migrator is ManagerProxyTarget, L2ArbitrumMessenger, IMigrator {
 
     event L1MigratorUpdate(address _l1Migrator);
 
+    event ProtocolContractUpdate(bytes32 _id, address _address);
```

Comment:
```suggestion
    event ControllerContractUpdate(bytes32 id, address addr);
```

- Match convention of no underscores in event params
- Align with naming of `syncControllerContracts()` helper

Reply:
Fixed in 01b3813